packages/zoe/src/issuerStorage.js

```diff
@@ -52,6 +53,7 @@ export const makeIssuerStorage = exportedIssuerStorage => {
    * @returns {IssuerRecord}
    */
   const storeIssuerRecord = issuerRecord => {
+    assertInstantiated();
```

Comment:
It's a shame that sprinkling `assertInstantiated()` into a bunch of methods is the best way to express this. But unfortunately it probably is. No change suggested.